src/uthash.h

```diff
@@ -593,17 +626,17 @@ do {
   /* Main loop */                                                                \
   for (;_sfh_len > 0U; _sfh_len--) {                                             \
     hashv    += get16bits (_sfh_key);                                            \
-    _sfh_tmp  = ((uint32_t)(get16bits (_sfh_key+2)) << 11) ^ hashv;              \
+    _sfh_tmp  = ((uthash_uint32_t)(get16bits (_sfh_key+2)) << 11) ^ hashv;       \
     hashv     = (hashv << 16) ^ _sfh_tmp;                                        \
-    _sfh_key += 2U*sizeof (uint16_t);                                            \
+    _sfh_key += 2U* 2/*sizeof(uint16_t)*/;                                       \
```

Comment:
There is this and another instance where I replaced sizeof(uint16_t) with 2. Now, AFAIU, sizeof would not have worked if uint16_t is not defined, so this should've failed on exotic platforms anyway. And sizeof(uint16_t) must always be 2. If the chosen get16bits() implementation is using byte assembling, then there is still no uint16_t (or uthash_uint16_t).